packages/react/src/components/FolderIcon/FolderIcon.tsx

```diff
@@ -0,0 +1,180 @@
+import { FolderIconProps } from './FolderIcon.types';
+
+import clsx from 'clsx';
+import { folderIconClasses, getFolderIconUtilityClass } from './FolderIcon.classes';
+
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+import { styled, useThemeProps } from '@mui/material/styles';
+import { capitalize } from '@mui/material/utils';
+
+import { FolderIconFilledIcon, FolderIconIcon, FolderIconOutlinedFilledIcon, FolderIconOutlinedIcon } from './icons';
+const SIZE = {
+  '100': {
+    width: '15',
+    height: '20',
+  },
+  '200': {
+    width: '18',
+    height: '24',
+  },
+  '300': {
+    width: '21',
+    height: '28',
+  },
+  '400': {
+    width: '24',
+    height: '32',
+  },
+  '500': {
+    width: '27',
+    height: '36',
+  },
+  '600': {
+    width: '30',
+    height: '40',
+  },
+  '700': {
+    width: '33',
+    height: '44',
+  },
+  '800': {
+    width: '36',
+    height: '48',
```

Comment:
Давай тут сделает в px, а в самих иконках вообще не будем явно из прописывать
```suggestion
    height: '48px',
```

В иконке сделать так
```jsx
<SvgIcon viewBox="0 0 36 48" {...props}>
```